lib/analytical/modules/google.rb

```diff
@@ -41,6 +41,13 @@ def track(*args) # page, options
         JS
       end
 
+      def identify(*args) # id, options
+        <<-JS.gsub(/^ {10}/, '')
+          ga('set', 'dimension3', id);
+          ga('send', 'pageview');
```

Comment:
This is already triggered in the javascript initialisation, so it's repeated here.
So maybe the dimension should also be set there (similar to the `userId`) and the identify is not needed.